src/component/Tabs.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Tabs = props => {
+  let { setCurrentView } = props;
+
+  return (
+    <div className="tab-container">
+      <div className="tab" onClick={() => setCurrentView('home')}>
```

Comment:
We have created constants for the views somewhere above. Let's use those constant here.
`'home'` -> `TABS.HOME`